src/Event.php

```diff
@@ -33,8 +34,23 @@ public function acknowledge(Message $message)
         array_map(function ($name) use ($message) {
             $this->emitter->emit($name, $message);
         }, [
-            static::QUEUE_ACKNOWLEDGE,
-            sprintf('%s.%s', static::QUEUE_ACKNOWLEDGE, $message->handler())
+            static::MESSAGE_ACKNOWLEDGE,
+            sprintf('%s.%s', static::MESSAGE_ACKNOWLEDGE, $message->handler())
```

Comment:
Seems like this could be abstracted to a helper:

```php
$this->eventName(static::MESSAGE_FOO, $message->handler());
```

Maybe over engineered?